web/src/features/fireWeather/components/graphs/WindGraph.tsx

```diff
@@ -142,7 +156,8 @@ const WindGraph = (props: Props) => {
           gdpsData.windSpdLine,
           rdpsData.windSpdLine,
           hrdpsData.windSpdLine,
-          observationData.windSpdLine
+          observationData.windSpdLine,
+          forecastData.windSpdLine
```

Comment:
Ordering actually matters here. By putting observation at last it will appear on top of all the graphics, so can you change the order here and down below for arrows??

Reply:
I also forgot to check for min and max values - I've added that too.